CMakeLists.txt

```diff
@@ -5,3 +5,4 @@
 zephyr_include_directories_ifdef(CONFIG_OPTEE_CLIENT public)
```

Comment:
In commit message: `TA load message.` -> `TA load request.`

Reply:
Done

Reply:
Still can see message instead of request